### DISCUSSION TITLE
Has anyone managed to get anon8231489123/vicuna-13b-GPTQ-4bit-128g running

### DISCUSSION THREAD

**Discussion by BennettFourr** [oobabooga/text-generation-webui#1257] [Q&A] [answered]
I have been trying to run anon8231489123/vicuna-13b-GPTQ-4bit-128g on windows for a while but cant get it to work. Has anyone got it running with text-generation-webui.

**Comment by GitHub1712** [comment]
Not tried but eachadea/vicuna-13b-1.1 is my new best friend.

**Comment by olihough86** [comment]
Yes but I'm not exactly sure how... I did have to trash my setup and start again also there was the whole thing with act-order I do remember there is a separate .pt that worked, only having one of models int he folder seemed to hep too, there is an uncensored 13b now too reeducator/vicuna-13b-free but it is 1.0 and keeps talking to its self

**Comment by Samjack1533** [comment]
Its works perfectly.Actually best model I use so far beside its lecture times

**Reply by BennettFourr** [reply]
How did you get it working

**Reply by Samjack1533** [reply]
What kind of error you get did you have a screenshot?

**Comment by ExtinctionEvent** [comment, marked as answer]
Yes. I  did a fresh install of ooba, and then followed the gptq installation guide on the fresh install. Then ran..load with #> python server.py --model vicuna-13b-GPTQ-4bit-128g --wbits 4 --groupsize 128

**Comment by GitHub1712** [comment]
Try to download with the download-model.bat script:
1. Install oobabooga/text-generation-webui
2. Start download-model.bat, tyle L for custom model, insert eachadea/vicuna-13b-1.1
After download you should be able to select the model from the web ui.
Alternative you could download using the web ui from the model tab.

The eachadea/vicuna-13b-1.1 performs very good for me. It is AI expert and knows itself. But it is censored and maybe 30B models should be better.
Someone running a 30B model? Which is cool?
